pkg/agent/secondarynetwork/init.go

```diff
@@ -48,13 +50,39 @@ func Initialize(
 	nodeName string,
 	podUpdateSubscriber channel.Subscriber,
 	stopCh <-chan struct{},
-	config *agentconfig.SecondaryNetworkConfig, ovsdb *ovsdb.OVSDB) error {
+	secNetConfig *agentconfig.SecondaryNetworkConfig, ovsdb *ovsdb.OVSDB) error {
 
-	ovsBridgeClient, err := createOVSBridge(config.OVSBridges, ovsdb)
+	ovsBridgeClient, err := createOVSBridge(secNetConfig.OVSBridges, ovsdb)
 	if err != nil {
 		return err
 	}
 
+	// we only support moving and restoring of interface configuration to OVS Bridge for single physical interface.
+	phyInterfaces := make([]string, len(secNetConfig.OVSBridges[0].PhysicalInterfaces))
```

Comment:
I meant the following:

```
	phyInterfaces := secNetConfig.OVSBridges[0].PhysicalInterfaces
	if len(phyInterfaces) == 1 {
        	phyInterfaces := make([]string, len(secNetConfig.OVSBridges[0].PhysicalInterfaces))
		copy(phyInterfaces, secNetConfig.OVSBridges[0].PhysicalInterfaces)
		ifaceName := phyInterfaces[0]
	...
```

But agree it is not a very useful optimization. So, up to you.